openmls/src/group/core_group/validation.rs

```diff
@@ -172,50 +171,6 @@ impl CoreGroup {
 
                 return Err(ProposalValidationError::InsufficientCapabilities);
             }
-
```

Comment:
There is neither a `capabilities` (nor `required_capabilities`) extension for key packages. Thus, I think this can go.